kernel_gateway/services/notebooks/handlers.py

```diff
@@ -35,6 +35,11 @@ def _request_assignment_for_lang(self, kernel_name, expression):
             statement = self._assignment_statements[None]
         return statement.format(expression)
 
+    def _request_json_for_lang(self, kernel_name, expression):
+        if kernel_name == 'julia-0.3':
+            expression = expression.replace('\"', '\\"')
```

Comment:
Instead of patching for Julia specifically, I think we should fix this in general. I suspect the number of languages that use double-quotes for strings instead of single quotes is going to be higher. So let's get code in that uses the double-quote syntax by default and properly escapes the JSON string quotes with backslashes.